recipes/centos_wildfly_swarm/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2012-2017 Red Hat, Inc
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+FROM registry.centos.org/centos/centos
+
+MAINTAINER Dharmit Shah
+
+ARG JAVA_VERSION=1.8.0
+ARG SWARM_VERSION=2017.3.2
+
+ENV JAVA_HOME=/usr/lib/jvm/java-${JAVA_VERSION} \
+    M2_HOME=/opt/rh/rh-maven33/root/usr/share/maven
+
+RUN yum -y update && \
+    yum -y install wget sudo openssh-server centos-release-scl && \
+    yum -y install rh-maven33 java-${JAVA_VERSION}-openjdk-devel && \
+    yum clean all && \
+    sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd && \
+    echo "user ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers && \
+    sed -i 's/Defaults    requiretty/#Defaults    requiretty/g' /etc/sudoers && \
```

Comment:
```
-    sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd && \
+    sed -ri 's/UsePAM yes/#UsePAM yes/g' /etc/ssh/sshd_config && \
+    sed -ri 's/#UsePAM no/UsePAM no/g' /etc/ssh/sshd_config && \
```
Can you please replace it this way? I have just discovered that without these lines sshing into a container using key is impossible.